### ISSUE TITLE
Genegraph-ui does not load data when using 'Geisingerized' computers

### ISSUE THREAD

**Issue by tnavatar** [clingen-data-model/genegraph#286]
Initial testing with users suggests the app is returning either a 200 or a 403 status code on connection to Genegraph. Since this connection ought to be happening via Websockets, neither of these is actually appropriate.

Will need to investigate falling back to an HTTPS connection when Websocket has errors like this.

**Comment by KelseaChang5** [comment]
December 7th 2021 Triage: Tristan will open ticket with Geisinger support and monitor. Keep our issue in backlog until something material develops that we can address.

**Comment by tnavatar** [comment]
Opened INC1753749 with Geisinger. This was specifically requested by the curators when we reviewed Genegraph with them on Monday, so it is now elevated in priority.

**Comment by tnavatar** [comment]
I looked into this issue after connecting to the Geisinger network using their VPN. The Javascript console returns an error message, as if the CORS policy was violated. I suspect that the HTTPS proxy for Geisinger is injecting a restrictive header into any of these requests. I submitted a request to see if an exception could be made for this site. The response I got is that Genegraph would be considered an 'external application' and therefore require IT approval before anyone would be willing to to anything to help. The approval process commonly takes months or years to complete, if it ever does.

In other words, the request to handle the Geisingerized computers has been Geisingerized. May need to find another way; possibly embedding the UI in Genegraph itself, so as to avoid the cross site issues

**Comment by tnavatar** [comment]
No longer pertinent. Closing.
